src/sentry/services/hybrid_cloud/rpc.py

```diff
@@ -350,6 +373,22 @@ def create_delegation(cls, use_test_client: bool | None = None) -> Self:
         return service  # type: ignore[return-value]
 
 
+def list_all_service_method_signatures() -> Iterable[RpcMethodSignature]:
+    """List signatures of all RPC methods in the global registry."""
+
+    from sentry.services import hybrid_cloud as hybrid_cloud_service_pkg
+
+    # Forcibly import all service packages to ensure the global registry is fully populated
+    for (importer, name, is_pkg) in pkgutil.iter_modules(hybrid_cloud_service_pkg.__path__):
+        if is_pkg:
+            module = importer.find_module(name)  # type: ignore[call-arg] mypy infers union but we only get one side of it
+            if module:
+                module.load_module(name)
```

Comment:
I'd recommend the simpler:

```python
for _, name, _ in pkgutil.iter_modules(hybrid_cloud_service_pkg.__path__, prefix=f'{hybrid_cloud_service_pkg.__name__}.'):
    __import__(name)
```

Reply:
though actually I think `walk_packages` is the one instead of `iter_modules`

Reply:
Updated to use walk_packages.